packages/backend/test/utils/workspaceFile.spec.ts

```diff
@@ -29,30 +29,43 @@ describe("extension unit test", () => {
   });
 
   describe("createWorkspaceFile", () => {
-    it("is not in BAS, workspace file does not exist", () => {
-      Constants.IS_IN_BAS = false;
-      const targetFolerPath = "targetFolerPath";
+    it("workspace file does not exist", () => {
+      const targetFolderPath = normalize(join(Constants.HOMEDIR_PROJECTS, "../tmp/targetFolerPath"));
       const expectedWsFilePath = join(Constants.HOMEDIR_PROJECTS, `workspace.code-workspace`);
       uriMock.expects("file").withArgs(expectedWsFilePath);
       fsMock.expects("existsSync").withArgs(expectedWsFilePath).returns(false);
-      fsMock.expects("writeFileSync").withArgs(expectedWsFilePath);
+      const fileContent = {
+        folders: [
+          {
+            path: `${relative(dirname(expectedWsFilePath), targetFolderPath)}`,
+          },
+        ],
+        settings: {},
+      };
+      fsMock.expects("writeFileSync").withArgs(expectedWsFilePath, JSON.stringify(fileContent));
 
-      WorkspaceFile.create(targetFolerPath);
+      WorkspaceFile.create(targetFolderPath);
     });
 
-    it("is not in BAS, workspace file exists", () => {
+    it("workspace file exists", () => {
       Constants.IS_IN_BAS = false;
```

Comment:
This line is redundant in new implementation

Reply:
done